### PR TITLE
Change HFP by HF references on the code

### PR DESCRIPTION
**Pull request by cazapatamar**

### CODE REVIEW

File: src/landscape/Landscape.jsx [hunk 1]
```diff
@@ -123,7 +123,7 @@ Landscape.propTypes = {
   biomas: PropTypes.array,
   distritos: PropTypes.array,
   currentHF: PropTypes.array,
-  currentHFPValue: PropTypes.number,
+  currentHFValue: PropTypes.number,
```
Comment (alegnaaived) [inline src/landscape/Landscape.jsx:126]: Could you please change it to hfCurrentValue?

File: src/landscape/Landscape.jsx [hunk 2]
```diff
@@ -96,7 +96,7 @@ class Landscape extends React.Component {
         component: (
           <HumanFootprint
             currentHF={currentHF}
-            currentHFPValue={currentHFPValue}
+            currentHFValue={currentHFValue}
             hfPersistence={hfPersistence}
             hfTimeline={hfTimeline}
             timelineHFArea={timelineHFArea}
```
Comment (alegnaaived) [inline src/landscape/Landscape.jsx:102]: It makes me think to change it too, to keep hfTimelineArea to standardize the beginning as "hf"

File: src/Search.jsx [hunk 3]
```diff
@@ -98,7 +98,7 @@ class Search extends Component {
   };
 
   /**
-   * Set in state timelineHFP area details for strategic ecosystems (SE) in the selected area
+   * Set in state timelineHF area details for strategic ecosystems (SE) in the selected area
```
Comment (alegnaaived) [inline src/Search.jsx:101]: Then, it should be better hfTimelineArea instead of timelineHF area. Thank you for change this

Comment (cazapatamar) [reply src/Search.jsx]: Done

**Pull request merged**